cluster-autoscaler/core/scale_down.go

```diff
@@ -1249,11 +1252,17 @@ func (sd *ScaleDown) deleteNode(node *apiv1.Node, pods []*apiv1.Pod, daemonSetPo
 	return status.NodeDeleteResult{ResultType: status.NodeDeleteOk}
 }
 
-func podsToEvict(pods []*apiv1.Pod, shouldEvict bool) []*apiv1.Pod {
-	if shouldEvict {
-		return pods
+func podsToEvict(pods []*apiv1.Pod, evictByDefault bool) (evictable []*apiv1.Pod) {
```

Comment:
Can you rename this to make it clear it's related to ds and not 'regular' pods? Also maybe consider moving it to cluster-autoscaler/utils/daemonset?

Reply:
I think moving this to `cluster-autoscaler/utils/daemonset` will suffice as a naming change, since it will then become `daemonset.PodsToEvict`. Will move it there.